rts/lra/pom.xml

```diff
@@ -36,7 +42,44 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>repo.eclipse.org-snapshots</id>
+            <name>Eclipse MicroProfile Repository - Snapshots</name>
+            <url>https://repo.eclipse.org/content/repositories/microprofile-snapshots</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
```

Comment:
probably not needed when LRA version is 1.0-RC1

Reply:
We will still need the snapshots as soon as we push changes in preparation for 1.0-RC2 or 1.0-Final